sqlparse/filters/aligned_indent.py

```diff
@@ -15,11 +15,12 @@ class AlignedIndentFilter(object):
     join_words = (r'((LEFT\s+|RIGHT\s+|FULL\s+)?'
                   r'(INNER\s+|OUTER\s+|STRAIGHT\s+)?|'
                   r'(CROSS\s+|NATURAL\s+)?)?JOIN\b')
+    by_words = ('GROUP BY', 'ORDER BY')
```

Comment:
```by_words``` is never used. What about making a regex of it (like it's done with join_words) and use it really the same? I think ```r'(GROUP|ORDER)\s+BY\b'``` should work.

Reply:
Addressed.